frontend/packages/hardhat/hardhat.config.ts

```diff
@@ -30,7 +30,7 @@ const config: HardhatUserConfig = {
       },
     },
   },
-  defaultNetwork: "localhost",
+  defaultNetwork: "sepolia",
```

Comment:
Usually you don't want to push this kind of changes | or create an environnement variable for it. So it won't disturb how people are using the frontend.